.github/workflows/test.yaml

```diff
@@ -0,0 +1,54 @@
+name: Test
+on:
+  push:
+    branches: [master, main]
```

Comment:
should we remove usage of the term `master`?

Reply:
yes. doing.